admin/client/src/components/Preview/Preview.js

```diff
@@ -7,25 +7,39 @@ import SilverStripeComponent from 'lib/SilverStripeComponent';
 class Preview extends SilverStripeComponent {
 
   render() {
-    return (
-      <div className="cms-content__right preview">
-        <iframe className="preview__iframe" src={this.props.previewUrl}></iframe>
+    // @todo - Multiple preview views with toggle slider
+    let body = null;
+    if (!this.props.previewUrl) {
+      body = (
         <div className="preview__overlay">
           <h3 className="preview__overlay-text">There is no preview available for this item.</h3>
         </div>
+      );
+    } else if (this.props.previewUrl.match(/\.((gif)|(jp(e?)g)|(png))($|\?)/i)) {
```

Comment:
Little bit ick. We can't pass through a preview Type arg or something? Or have a special handler?